CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -0,0 +1,507 @@
+- time - Android Studio setup is sadly complicated
+  - Multiple hours are typical, though mostly spent on waiting for various downloads
+  - setting up Android Studio will likely be more complex than writing the code
+- [basic ability to use git](https://git-scm.com/docs/user-manual) (but as mentioned, anything GitHub specific will be explained)
+  - there is version control integration available in Android Studio
+    - From the Android Studio **VCS** menu, click **Enable Version Control** Integration ([see official docs](https://developer.android.com/studio/intro#version_control_basics))
+    - Go to File -> Settings -> Version control in Android Studio to optionally configure git integration
+  - Using git from command line or terminal in Android Studio also works fine
+- GitHub [account](https://github.com/signup)
+- around 15GB of free space on disk
+  - the initial setup will typically download half of the Internet while getting emulators and dependencies
+
+# Dependencies - initial setup
+
+- install necessary software ([Android Studio](https://developer.android.com/studio) and [git](https://git-scm.com/downloads))
+- create a GitHub [account](https://github.com/signup) if needed
+- visit [https://github.com/streetcomplete/StreetComplete](https://github.com/streetcomplete/StreetComplete) and press the "fork" button on the top right
+	+ this creates a copy of StreetComplete repository that you control and can prepare code there
+- clone your fork of a StreetComplete repository
+- open StreetComplete in Android Studio
+- [setup an emulator in Android Studio](https://developer.android.com/studio/run/emulator#install) (you can also [connect to a real device via USB](https://developer.android.com/studio/run/device), this can replace using the emulator)
+- run StreetComplete in the emulator (to verify that everything was setup as required)
+
+If you are doing it for the first time, don't worry if there is an error to solve along the way, this is typical for setting up an Android development environment. See [CONTRIBUTING file](CONTRIBUTING.md#development) which has some links to information about the setup.
+
+# Invent a new quest
+
+## Own ideas
+To [repeat](CONTRIBUTING.md#developing-new-quests) from that documentation file:  [**open an issue** discussing the quest](CONTRIBUTING.md#developing-new-quests), before starting other work. This way it can be confirmed that such quest can be included. This can be skipped if you are an [experienced](https://github.com/streetcomplete/StreetComplete/discussions/3450) StreetComplete contributor.
+
+## Existing proposals
+You can also look at [quest proposals waiting for implementation](https://github.com/streetcomplete/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+quest%22).
```

Comment:
Both as a general question and a comment on the doc: Does "quest proposals waiting for implementation" mean that every issue that has the label "New Quest" is meant to be viewed as "approved as okay and just waiting to be implemented"? (obviously excluding issues where additionally "Blocked" is applied) 
On that note, maybe the link could be extended to exclude issues with that label.
```suggestion
You can also look at [quest proposals waiting for implementation](https://github.com/streetcomplete/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+quest%22+-label%3A%22blocked%22).
```

Reply:
> Does "quest proposals waiting for implementation" mean that every issue that has the label "New Quest" is meant to be viewed as "approved as okay and just waiting to be implemented"?

yes - is it necessary to make it more clear?

> blocked

well, blocked ones can progress by submitting work upstream.

Not sure is it necessary to exclude them. Some of it is easier than actually making SC quest

Reply:
Ah ok, I always thought that the new quest label simply means "This issue proposes a new quest" rather than "This issue proposes a new quest that was already determined to be a good fit and can be implemented"
> well, blocked ones can progress by submitting work upstream.
Oh well, that shows another misunderstanding on my side of the meaning of the issue lables

Maybe (if you deem this to be a misunderstanding more people could experience) a section could be added explaining what the different lables mean.

Reply:
I added descriptions to the labels themselves as a first measure: https://github.com/streetcomplete/StreetComplete/labels

Reply:
I opened https://github.com/streetcomplete/StreetComplete/discussions/3840